### DISCUSSION TITLE
My GitHub Education application was verified as Student not Teacher

### DISCUSSION THREAD

**Discussion by eeholmes** [community/community#50277] [GitHub Education] [answered]
I was verified fine, but as a student. I am the instructor and wanted to use GitHub for teaching my class. I am almost positive but I clicked the teacher button but perhaps not. The student benefits are not what I need. Any human or bot avenue for fixing this?

**Comment by Anmol-Baranwal** [comment]
If you have verified your account as a student but need access to the teacher benefits for teaching your class on GitHub, you can contact GitHub support for assistance [here](https://support.github.com/request?tags=docs-generic)

Alternatively, you can try re-verifying your account as a teacher. To do this, go to the GitHub Education page and click on the "Get benefits as a teacher" button. From there, follow the instructions to verify your teacher status. Once verified, you should have access to the teacher benefits on GitHub. (Not sure)

**Reply by Anmol-Baranwal** [reply]
@eeholmes 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by eeholmes** [comment, marked as answer]
@Anmol-Baranwal Hi, the link for GitHub support didn't help so much. I was there before and it's not clear how to get help on this issue.

I think I will have the other instructors apply and maybe one of them can get the Team benefit for our class. I cannot reload my instructor ID because GitHub recognizes that I have used this image already.

**Reply by Anmol-Baranwal** [reply]
Okay, thanks for letting me know.

You can follow these steps as shown in the image to generate a ticket:

1. GitHub Education program or benefits
![](https://user-images.githubusercontent.com/74038190/226400873-c996c637-ede9-43c8-b0fc-fde6e668a8a9.png)

2. Contact us
![Screenshot (3096)](https://user-images.githubusercontent.com/74038190/226400876-c56a32bf-4857-4bbf-b551-fdf6940d58a4.png)

3. Other support
![Screenshot (3097)](https://user-images.githubusercontent.com/74038190/226400866-f16b96c1-85ce-4ae1-b8d8-426c1a2707b8.png)

4. You can now generate a ticket
![image](https://user-images.githubusercontent.com/74038190/226401635-8464649d-2f7b-4dc6-8bdc-5a098c813e7d.png)

<br>

Please let me know if you encounter any further problems. I will gladly address them.

**Comment by eeholmes** [comment]
Thanks! I put in a ticket but alas it just told me to look at my dashboard and click on "organizations" to upgrade. It didn't address the issue which is that the GitHub Education system thinks I am a student not a faculty.

Oh well, thanks for your help.

**Comment by WanYakuwea** [comment]
Getting in touch with their support team will give you the best chance of rectifying the issue and ensuring you have the necessary tools and features for teaching your class effectively.

**Comment by gnaist** [comment]
I am also having this same issue. Were you able to resolve this issue? Did contacting support help?

**Reply by samus-aran** [reply]
We've commented on[ this discussion](https://github.com/orgs/community/discussions/77828) to progress your request 👍

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still require assistance please feel free to re-open.
